joomla/joomla-cms
# [4.0]User - User Actions Log - Need Tooltip for search field #26659

### Summary of Changes
Pull Request for Issue #[#26659 ] - User - User Actions Log - Need Tooltip for search field


### Testing Instructions
Go to Administrator Login Page
Click on Users -> User Actions Log
Check Search input box tooltip

### Expected result

After hovering on the search field tooltip should display.

### Actual result

![Screenshot from 2019-10-19 16-35-26](https://user-images.githubusercontent.com/17097405/67143934-80b2f380-f28e-11e9-936a-646115614e3e.png)


### Documentation Changes Required
